### PR TITLE
Add DhoklaStellar Fafda

### PR DESCRIPTION
**Pull request by akshar314**

### CODE REVIEW

**Comment by akshar314** [comment]
Please review this, @casey-collab

File: Gujarati/DhoklaStellarFafda~/CS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//akshar314
```
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by akshar314** [comment]
Done!

File: Gujarati/DhoklaStellarFafda~/CS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Are we rushing in or going sneaky beaky like...
```
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by akshar314** [comment]
@casey-collab!

File: english/marshmallow.md [hunk 3]
```diff
@@ -934,4 +934,6 @@ You hear a hollow voice say "Plugh".  Mesmerized, you say "[Plugh](plugh/plugh.m
 
 A light appears all around you and something comes into focus in the distance. [What is that?](disney/disney.md)
 
-Story about Indian Hockey Player [Rupinder Pal Singh](Rupinderpal/rupinder.md)
\ No newline at end of file
+Story about Indian Hockey Player [Rupinder Pal Singh](Rupinderpal/rupinder.md)
+
+yo [cs](DhoklaStellarFafda~/CS.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Gujarati/DhoklaStellarFafda~/CS.md (hunk 2 above)
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 3 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Gujarati/DhoklaStellarFafda~/CS.md (hunk 2 above)
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 3 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Gujarati/DhoklaStellarFafda~/CS.md (hunk 2 above)
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 3 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Gujarati/DhoklaStellarFafda~/CS.md (hunk 2 above)
Comment (casey-collab) [inline Gujarati/DhoklaStellarFafda~/CS.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 3 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by akshar314** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 3 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by akshar314** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by akshar314** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**